src/components/editor/menu/DiagramOptions.tsx

```diff
@@ -36,7 +38,13 @@ const DiagramOptions = ({
             <TableChartIcon />
           </IconButton>
         )}
-        {onCutSetAnalysis && <Button onClick={onCutSetAnalysis}>cutsets</Button>}
+        {onCutSetAnalysis && (
+          <Tooltip title="Cutset">
```

Comment:
I am not sure, but I think we are using i18n?

Reply:
@blcham should we translate it in CZ ? If so, is "řezy" correct word here ?

Reply:
![image](https://github.com/akaene/fta-fmea-ui/assets/3962237/69aea4c4-4e7f-48fd-b562-1f5dafeb99fe)


Cutset --> řez
Cutsets --> řezy

Reply:
Added "Switch to cutsets view" and "Přepnout do pohledu řezu"